lib/xandra/protocol.ex

```diff
@@ -456,37 +472,37 @@ defmodule Xandra.Protocol do
     message
   end
 
-  @spec decode_response(Frame.t(:error), term) :: Error.t
+  @spec decode_response(Frame.t(:error), any) :: Error.t
```

Comment:
Do we use `any` in the codebase? #lazyweb

Reply:
Seems no, will change back. 🤷‍♂️